source/ed/types.cpp

```diff
@@ -124,6 +124,15 @@ bool VehicleJourney::operator<(const VehicleJourney& other) const {
     return this->uri < other.uri;
 }
 
+bool VehicleJourney::has_prev_vj_prolongation_on_same_stop_point() const {
```

Comment:
Given the different use, I had trouble understanding (why the prev_vj was re-checked and so on).
Maybe change it to a method returning enum and adapt use?
```suggestion
enum class StayInStatus {
    NoStayIn = 0,
    SameStop,
    DifferentStop
};

StayInStatus VehicleJourney::get_prev_vj_stay_in_status() const {
```